media/js/firefox/whatsnew/whatsnew-69.js

```diff
@@ -5,6 +5,27 @@
 (function() {
     'use strict';
 
+    // Issue #7671 measure potential UITour failure for bug 1557153
+    function pingUITour() {
+        var startTime = performance.now();
+        var timeout = window.setTimeout(trackImpression, 1000, 'UITour Timeout');
+
+        Mozilla.UITour.ping(function() {
+            window.clearTimeout(timeout);
+            trackImpression('UITour Response');
+        });
+
+        function trackImpression(eLabel) {
+            var endTime = performance.now() - startTime;
+
+            window.dataLayer.push({
```

Comment:
@hoosteeno can you advise if this `dataLayer` event looks ok?

Reply:
Yes, thank you.